docs/source/install.rst

```diff
@@ -0,0 +1,69 @@
+.. _install-guilde:
+
+Installation Guide
+==================
+
+Dependencies
+------------
+
+ChainerUI is developed under Python 2.7+, 3.5+, 3.6+. For other requirement, see ``requirements.txt``.
+
+.. literalinclude:: ../../requirements.txt
+  :caption: requirements.txt
+
+ChainerUI uses ``sqlite3`` module which is included in the standard Python library. If Python is built from source, need to enable ``sqlite3`` module before building Python.
```

Comment:
ChainerUI uses the... from source, the sqlite3 module must be installed before building Python.